.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+### Step 0: Are you in the right place?
```

Comment:
Do you want to add the metadata to the top of this file?

Example:
https://raw.githubusercontent.com/google/ExoPlayer/release-v2/.github/ISSUE_TEMPLATE/bug.md